### ISSUE TITLE
Jump to line on click lint error

### ISSUE THREAD

**Issue by mjgpy3** [rhysd/Shiba#22]
I'm loving this tool so far :+1: :+1: 

On my machine I cannot see the line numbers of the lint errors:

![image](https://cloud.githubusercontent.com/assets/1709849/13188281/822f2060-d71e-11e5-8d38-336872b2ed33.png)

It would be nice if it jumped to or highlighted the line and/or jumped to it.

**Comment by rhysd** [comment]
Yeah I agree.  Currently it's hard (but not impossible) to implement it because there is no mapping from markdown document to converted HTML document  (which HTML element corresponds to specific markdown element?).
We should introduce markdown AST for this feature and I plan to do that for differential DOM update.  So I will remain this issue open although it may take long time.
